client/my-sites/plans-features-main/index.tsx

```diff
@@ -61,6 +61,8 @@ import type { IAppState } from 'calypso/state/types';
 
 import './style.scss';
 
+const SPOTLIGHT_VISIBLE_PLANS_COUNT = 6;
```

Comment:
Why 6? And shouldn't this be targeted on plans intent instead? 

I think encoding a number this way will propagate with undesired results soon e.g. Woo plans have a plan spotlight with only 3 plans (and they are on the road to aligning with the rest of the framework)

Reply:
If we want to show a spotlight for a smaller grid, we need to iterate and come up with a design that suites a smaller grid.

Reply:
Right. I'm not arguing against that, but hardcoding a design effect this way is a little heavy-handed. You can target using the intent flag to only enable the spotlight for the wpcom-default intent and let other use cases opt in/out. Spotlight with fewer plans has been working for Woo plans for some time now.

Reply:
Ah so @chriskmnds what you're saying is:

```javascript
const isSpotlightOnCurrentPlanAllowed = intent === null
```

If I understand this correctly?

Reply:
Got it @chriskmnds 💯, that seems like a good alternative too. 
Thanks for laying out the exact implementation 🤟 😄 .

Reply:
Fixed by this change https://github.com/Automattic/wp-calypso/pull/80232/commits/6fca78c3e1023372d8e1324b9b5479fa0018bcc9